pkg/apis/vpc/v1alpha1/subnet_types.go

```diff
@@ -103,6 +104,17 @@ type SubnetList struct {
 	Items           []Subnet `json:"items"`
 }
 
+type SubnetAdvancedConfig struct {
+	// Connectivity status of the Subnet from other Subnets to the VPC.
+	// Default value is "Connected".
+	// +kubebuilder:validation:Enum=Connected;Disconnected
+	// +kubebuilder:default=Connected
+	ConnectivityState string `json:"connectivityState,omitempty"`
```

Comment:
Should we define a type for ConnectivityState? @tnqn

Reply:
thanks, I defined type of ConnectivityState.